### PR TITLE
Remove dependency on `paket`

### PR DESCRIPTION
**Pull request by razzmatazz**
This is a proposal PR that removes dependency on `paket` -- sorry I could not get tests to work in #7 and `paket` IMHO adds to much complexity for newbie F# dev like me..

freel free to shoot it down, it *looked* to make sense for me but not necessarily..

### CODE REVIEW

**Comment by baronfel** [comment]
That's fine by me, this repo is very small and doesn't have multiple projects, which is where paket starts to shine. I'd suggest using a git subrepo or something for the FsLibLog dependency instead of directly embedding it, though, because at least a git subrepo tracks the commit state if/when that file updates.

**Comment by Krzysztof-Cieslak** [comment]
Dropping Paket here seems Fine for me if we can get FsLibLog in some not-more-complex-than-Paket way ;-)

**Comment by baronfel** [comment]
@Krzysztof-Cieslak I was actually thinking of making another small msbuild project to replicate the github-file features in paket for partially this reason.

mini-spec:
* `<GithubFile Include="relative path from repo root" Repo="<repo shorthand>" Ref="git ref/tag/sha/branch" LocalPath="file will be downloaded to obj directory, this optional property determines what the file looks like in IDEs" />`
* element allows an optional set of `<Replace Match="source string" Replacement="destination string" />` that can go inside the xml element as children
* a restore-time task that 
  * downloads that file to the obj directory
  * runs any replacements on it
  * creates a marker file used for fast-up-to-date-checks
  * creates a `Compile` project item, so that the file is included in the build
  * adds both of these files to the `FileWrites` msbuild property so that `clean` works as you'd expect.

outstanding question is how to handle github auth, probably just require an ambient token for v1, etc.

**Comment by artempyanykh** [comment]
My 2¢: having a regular dotnet non-packet build would be great, because then it'd be much easier to vendor this lib as a git submodule, and hence easier to make changes/improvements, and contribute them upstream.

What I'm doing now is I've copy-pasted the LanguageServerProtocol.fs and FsLogLib.fs directly into my project as the current master version lacks some parts of the protocol that I need.

As a side question: what's so bad about just manually copying FsLogLib into this repo. Looking at the project releases, there's not that much traffic going on; so manually updating this single file doesn't seem like a big deal, given that it'd seriously simplify the project setup.

**Comment by baronfel** [comment]
It's probably not the worst thing in the world to manually copy FsLibLog.

**Comment by Krzysztof-Cieslak** [comment]
@baronfel You have just broken my heart :P 

But sure, if y'all think this is the way to move forward, please go ahead.

**Comment by razzmatazz** [comment]
OK, I have updated this PR by rebasing on top of `main` and removing conflicts

**Comment by baronfel** [comment]
Versioning should remain using the CHANGELOG.md mechanism that Ionide.KeepAChangelog enables. We shouldn't need to information track that in a Directory.Build.props.

File: src/LanguageServerProtocol.fs [hunk 1]
```diff
@@ -3000,7 +3000,7 @@ module Server =
 module Client =
     open System
     open System.IO
-    open LanguageServerProtocol.Logging
+    open FsLibLog
```
Comment (baronfel) [inline src/LanguageServerProtocol.fs:3003]: We should rename this namespace back to LanguageServerProtocol.Logging here and in the FsLibLog.fs file, to maintain the API surface.

**Comment by razzmatazz** [comment]
@baronfel ok, how does it look now?

**Comment by baronfel** [comment]
Looks great! I made a minor packaging change that I thought of during review - pinning the FSharp.Core reference in this repo to 6.0.0 so that the generated package dependency is `>= 6.0.0`, so consumers don't have to be on the latest and greatest 6.0 SDK to use this package.

**Pull request merged**